README.md

```diff
@@ -142,9 +150,13 @@ Submits the tar to the of-builder then configures an OpenFaaS deployment based u
 
 Writes statuses to GitHub Checks API showing build status and URLs for endpoints
 
+* Function: gitlab-status
```

Comment:
Are you adding gitlab-status in this PR? If not don't add it to the README

Reply:
Removed